lib/install.ps1

```diff
@@ -34,6 +34,17 @@ function install_app($app, $architecture, $global, $suggested, $use_cache = $tru
         return
     }
 
+    # Change 'innosetup' to 'installer.type:inno'
+    if ($manifest.innosetup) {
```

Comment:
Add deprecation message for innosetup property?

Reply:
Added.